src/index.html

```diff
@@ -1,12 +1,345 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500;700&display=swap&subset=cyrillic" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="images/logo.png" alt="logo image">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#laptops_and_computers"
+              >
+                Laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div data-qa="card" class="card">
+        <img
+            src="./images/imac.jpeg"
+            alt="IMac photo"
+            class="card__image"
+          >
```

Comment:
Fix all similar cases
```suggestion
        <img
          src="./images/imac.jpeg"
          alt="IMac photo"
          class="card__image"
        >
```